src/main/java/com/massivecraft/factions/listeners/FactionsBlockListener.java

```diff
@@ -187,6 +187,12 @@ public static boolean playerCanBuildDestroyBlock(Player player, Location locatio
             return true;
         }
 
+        if (action.equals("destroy") && Conf.enableUnprotectedBlocks) {
```

Comment:
should be `action == Action.DESTROY`

Reply:
*action* is a string? It can't be compared to enum destroy, unless I change it to *Action.fromString(action)*

Reply:
I am probably completely wrong

Reply:
oh, didn't read up. Assumed that `action` was an enum. But in that case, should equalsIgnore case.